packages/wallet/src/services/DelegationTracker/RewardAccounts.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable unicorn/no-nested-ternary */
 import { BigIntMath, deepEquals, isNotNil } from '@cardano-sdk/util';
-import { Cardano, RewardsProvider } from '@cardano-sdk/core';
+import { Cardano, RewardsProvider, StakePoolProvider } from '@cardano-sdk/core';
 import { Delegatee, RewardAccount, StakeKeyStatus } from '../types';
 import { KeyValueStore } from '../../persistence';
 import { Observable, combineLatest, concat, distinctUntilChanged, filter, map, merge, of, switchMap, tap } from 'rxjs';
+import { PAGE_SIZE } from '../TransactionsTracker';
```

Comment:
We could move this constant to a more commonplace since it's not transaction related.